src/path-planner.cc

```diff
@@ -138,6 +156,7 @@ namespace hpp {
 	   itn != roadmap ()->goalNodes ().end (); ++itn) {
 	ConfigurationPtr_t q1 ((initNode)->configuration ());
 	ConfigurationPtr_t q2 ((*itn)->configuration ());
+	assert (*q1 != *q2);
```

Comment:
Did you have problem with this ?
I think this is a bad idea. I think when the planner is asked to go from qinit to qgoal = qinit, it should return a path of length zero and not crash.

The problem is this should be handled by the steering method...

Reply:
Yes, I think it is better.

Reply:
Although it is questionable. The best would be to let the steering method generates it...

Reply:
I removed the assert and changed the kindoynamic steering-method to correctly return a path of length 0 in this case. 
I tested it with all steering-method available and it work correctly except for 'Hermite' which return a path of length 1 but without any motion.
Should I add a test in tryDirectPath anyway ?

Reply:
Thank you for having tested all steering methods. No need to add a test.